Editor/AGS.Editor/Resources/agsdefns.sh

```diff
@@ -1609,6 +1620,42 @@ builtin managed struct Region {
   int reserved[2];   // $AUTOCOMPLETEIGNORE$
 };
 
+#ifdef SCRIPT_API_v400
+builtin managed struct WalkableArea {
+  /// Gets the walkable area at the specified position on the screen.
+  import static WalkableArea* GetAtScreenXY(int x, int y);    // $AUTOCOMPLETESTATICONLY$
+  /// Returns the walkable area at the specified position within this room.
+  import static WalkableArea* GetAtRoomXY(int x, int y);    // $AUTOCOMPLETESTATICONLY$
+  /// Gets the drawing surface for the 8-bit walkable mask
+  import static DrawingSurface* GetDrawingSurface();  // $AUTOCOMPLETESTATICONLY$
+  /// Changes this walkable area's scaling level.
+  import void SetScaling(int min, int max);
+  /// Gets the ID number for this area.
+  import readonly attribute int ID;
+  /// Gets/sets whether this walkable area is enabled.

```

Comment:
`Gets/sets` in the comment should just be Gets for `ScalingMin` and `ScalingMax`